libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -53,7 +53,7 @@ const getSwapAPIVersion: () => number = () => {
   return version;
 };
 
-const ftx: SwapProviderConfig = {
+const ftx: ProviderConfig = {
   ...createExchangeProviderNameAndSignature({
     name: "FTX",
```

Comment:
Can we remove FTX stuff? @cjordan-ledger @sarneijim . Looping in @chabroA @JunichiSugiura for context